src/bot.ts

```diff
@@ -59,13 +59,16 @@ export class ChatGPTBot {
     return text;
   }
   async getGPTMessage(text: string, talkerId: string): Promise<string> {
-    return await this.chatGPTPool.sendMessage(text, talkerId);
+    let ret = await this.chatGPTPool.sendMessage(text, talkerId);
+    console.log(`chatgpt return: ${ret}`);
+    return ret;
   }
   // The message is segmented according to its size
   async trySay(
     talker: RoomInterface | ContactInterface,
     mesasge: string
   ): Promise<void> {
+    console.log(`trySay: ${mesasge}`);
```

Comment:
Remove debug messages?

Reply:
yeah sure, it's very unpolished. feel free to commit, i'm currently unavailable :(